tests/cmake_name_conflict/enc/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+add_custom_command(
+    OUTPUT
+    name_conflict_t.h
+    name_conflict_t.c
+    name_conflict_args.h
+    COMMAND edger8r --trusted --search-path ${CMAKE_CURRENT_SOURCE_DIR}/.. name_conflict.edl)
```

Comment:
You probably also want something like `DEPENDS edger8r name_conflict.edl` so that your dependencies are specified correctly.

Reply:
Thanks for the suggestion, please take a look on the updated patch.

Reply:
@andschwa Please check again when you got chance. Thanks!

Reply:
Hm, it's still not depending on the `edger8r` target, which you want in case it gets rebuilt. See other CMake uses of `edger8r` for the pattern.

Reply:
Just make sure---(in current patch) DEPENDS ../name_conflict.edl --- is not sufficient?

Reply:
Correct. That will trigger your bindings to be regenerated if the EDL file is changed, but not if the `edger8r` tool is rebuilt.

[Here](https://github.com/openenclave/openenclave/blob/71edd2cf92f52017410d5e3262628b962067560a/tests/oeedger8r/host/CMakeLists.txt#L7) is an example with both dependencies specified.

Reply:
Got it. Thanks! Just update the patch.